### PR TITLE
Jd/moi julia v0.7

### PR DESCRIPTION
**Pull request by jd-lara**
Enables parsing of matpower files from Julia v0.7. It will still fail for the `RTS - GLMC` data since the fix for #316 hasn't been implemented in the MOI branch.

### CODE REVIEW

File: REQUIRE [hunk 1]
```diff
@@ -1,6 +1,8 @@
-julia 0.6
+julia 0.7-
 JSON 0.6.0
-JuMP 0.17
+JuMP
```
Comment (ccoffrin) [inline REQUIRE]: min bound should be explicit here.

File: src/io/matpower.jl [hunk 2]
```diff
@@ -557,7 +557,7 @@ function add_dcline_costs(data::Dict{String,Any})
                     "startup" => 0.0,
                     "shutdown" => 0.0,
                     "ncost" => 2,
-                    "cost" => [0.0, 0.0, 0.0, 0.0]
+                    "cost" => [dcline["pminf"], 0.0, dcline["pmaxf"], 0.0]
```
Comment (ccoffrin) [inline src/io/matpower.jl]: This fix belongs in PowerModels master.

File: src/io/matpower.jl [hunk 3]
```diff
@@ -546,7 +546,7 @@ end
 
 "adds dcline costs, if gen costs exist"
 function add_dcline_costs(data::Dict{String,Any})
-    if length(data["gencost"]) > 0 && length(data["dclinecost"]) <= 0
+    if length(data["gencost"]) > 0 && length(data["dclinecost"]) <= 0 && length(data["dcline"]) > 0
```
Comment (ccoffrin) [inline src/io/matpower.jl]: This fix belongs in PowerModels master.

File: test/data/matpower/case5_pwlc.m [hunk 4]
```diff
@@ -52,3 +52,7 @@
 	4	 10	 0.00297	 0.0297	 0.00674	 240.0	 240.0	 240.0	 0.0	 0.0	 1	 -30.0	 30.0;
 ];
 
+mpc.dcline = [
+	1	 10	 1	 10	 10	 25.91	 -4.16	 1.1	 0.92617	 10	 900 -900 900 -900 900	 0	 0	 0	 0	 0	 0	 0	 0
+]
```
Comment (ccoffrin) [inline test/data/matpower/case5_pwlc.m]: This fix belongs in PowerModels master.

File: REQUIRE [hunk 5]
```diff
@@ -1,6 +1,8 @@
...
 InfrastructureModels 0.0.6 0.2-
 Compat 0.26.0
 Memento 0.6
+MathOptInterface
```
Comment (ccoffrin) [inline REQUIRE]: This should specify a min bound.

File: src/io/matpower.jl (hunk 3 above)
Comment (jd-lara) [reply src/io/matpower.jl]: I did a cherrypick to update these lines to the current version in master and implement the fix for PWL costs in DC-lines. How would you prefer to update these files in the `moi-julia-v0.7`?

**Pull request merged**